libraries/Microsoft.Bot.Builder.Teams/TeamsActivityHandler.cs

```diff
@@ -0,0 +1,306 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Schema;
+using Microsoft.Bot.Schema.Teams;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.BotBuilderSamples
```

Comment:
Should this be Microsoft.Bot.Builder.Teams

Reply:
nice catch! updated in the SDK code - but I'm having trouble with the Scenaio.sln - perhaps because I created that in vs19 and I'm trying to fix it in vs17 on my laptop - anyhow SDK code should be good.

Reply:
I also had that issue... which is what this was about:

https://github.com/microsoft/botbuilder-dotnet/commit/44526fc2e7890fee315e673e6b97d82c460dcadf

But i reverted it, thinking it might have been my env